.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+# Adapted form https://github.com/avsm/hello-world-action-ocaml/blob/master/.github/workflows/workflow.yml
+name: build
+
+on:
+  - pull_request
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          # - macos-latest
+          # - windows-latest
+        ocaml-version:
+          - 4.11.0
```

Comment:
If you want to pull the latest OCaml version, you can do so by specifying `4.11.x`. (With the naughty exception of 4.02.2, point releases are meant to be strictly compatible)

Reply:
Done!